src/main/java/com/iexec/sms/secret/SecretController.java

```diff
@@ -77,6 +80,8 @@ public ResponseEntity<Web3Secret> getWeb3Secret(@RequestHeader("Authorization")
     public ResponseEntity<String> addWeb3Secret(@RequestHeader("Authorization") String authorization,
                                                 @RequestParam String secretAddress,
                                                 @RequestBody String secretValue) {
+        secretAddress = Keys.toChecksumAddress(secretAddress);
```

Comment:
So we first checksum addresses then lowercase them in database (mentioning web2 and web3 endpoints right now)?

Reply:
Maybe to check with Pierre, but I understood it was a better idea to work with checksummed addresses.
And right now, I don't know how to address this without taking the risk of breaking things.